### PR TITLE
Add SLO rules for Loki AlertingRule and RecordingRule CRs

### PR DESCRIPTION
**Pull request by periklis**

### CODE REVIEW

File: resources/services/observatorium-template.yaml [hunk 1]
```diff
@@ -158,6 +158,67 @@ objects:
...
+      - alert: NeverFiringAlert
+        annotations:
+          description: Does not fire!
+          message: Alert not fired.
+        expr: 1 < 0
+        for: 1m
+        labels:
+          severity: page
+          source: alertingrule.loki.grafana.com
+    tenantID: rhobs
```
Comment (saswatamcode) [inline resources/services/observatorium-template.yaml:199]: I guess this `tenantID` is specific to Loki?

**Review by saswatamcode — APPROVED** [review]
LGTM!

One small question!

File: resources/services/observatorium-template.yaml (hunk 1 above)
Comment (periklis) [reply resources/services/observatorium-template.yaml:199]: Yes this the only thing it deviates from PrometheusRule. In general we introduced only tenant-aware CRDs in the Loki Operator. This operator deploys Loki with multi-tenancy by default. Of course you can use a single fake ID for putting anything into a single spot

Comment (saswatamcode) [reply resources/services/observatorium-template.yaml:199]: Ack Thanks!

**Review by saswatamcode — APPROVED** [review]

**Pull request merged**